src/widgets/PassageAncestorsWidget.vue

```diff
@@ -33,7 +33,7 @@
         return this.passage
           ? gql`
             {
-              passageLines(reference: "${this.passage.absolute}") {
+              passageTextParts(reference: "${this.passage.absolute}") {
```

Comment:
`ancestorsLens` still references `passageLines` below

Reply:
achh you are right, I was sure I had also updated that... fixing